lib/core/res/errors.dart

```diff
@@ -1,2 +1,14 @@
 // Common Errors
 class NetworkException implements Exception {}
+
+class ValidationException extends Error {
```

Comment:
should implement exception
```suggestion
class ValidationException impleents Exception {
```